### ISSUE TITLE
A problem has happened with **WidgetSlider**

### ISSUE THREAD

**Issue by Panda98277** [maruohon/malilib#132]
![image](https://github.com/maruohon/malilib/assets/118618400/949dfd6d-b233-40e6-b799-c05e301c4665)

maybe it's a bug

**Comment by vacla** [comment]
@Panda98277 it might be best to join the discord for dev support for malilib instead of making issues here on github

**Comment by Panda98277** [comment]
Thanks for your advice, I'm going to be a remember of it

**Comment by maruohon** [comment]
yeah dev support would be much preferred on the Discord server, so that I don't get an email notification for every message... :pain:

But this to me looks like you have such a wide slider that the texture is not wide enough. And I guess at least the old code didn't tile the texture if it isn't wide enough.

**Comment by Panda98277** [comment]
how do i join  the discord for dev support for malilib

**Comment by vacla** [comment]
You can join it thru https://masa.dy.fi/discord
